index.html

```diff
@@ -11,6 +11,7 @@
         <link href="css/ytplayer.css" rel="stylesheet" type="text/css" media="all" />
         <link href="css/theme.css" rel="stylesheet" type="text/css" media="all" />
         <link href="css/custom.css" rel="stylesheet" type="text/css" media="all" />
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
@saarthakchats  Don't use external libraries for just two icons. This makes site loading slower. Either save icon as images or use the already present sample icon for facebook and twitter.

Reply:
other participants can add icons for other social media also

Reply:
Thats why i left at 2 icons